dokka-runners/dokka-gradle-plugin/src/main/kotlin/dokka/DokkaPublication.kt

```diff
@@ -63,4 +64,7 @@ constructor(
 
     // TODO probably not needed any more, since Dokka Generator now runs in an isolated JVM process
```

Comment:
Q: just in case, does this `TODO` tracked via some issue? the same for other `TODO` left after merging dokkatoo - would be nice to have them tracked somehow

Reply:
Agreed, the TODOs should be removed or added in YouTrack. There are a few others scattered around.